src/main/java/uk/gov/hmcts/reform/finrem/ccddatamigration/service/GeneralMigrationService.java

```diff
@@ -77,17 +77,20 @@ public void processAllTheCases(final String userToken, final String s2sToken, fi
 
     private static boolean isCandidateForMigration(final CaseDetails caseDetails) {
         if (caseDetails != null && caseDetails.getData() != null) {
-            boolean caseNeedsMigration = false;
             Map<String, Object> caseData = caseDetails.getData();
-            if (caseData.get("natureOfApplication2") != null) {
-                List<String> list = (List) caseData.get("natureOfApplication2");
-                caseNeedsMigration = list.contains("Property Adjustment  Order");
-            }
-            return caseNeedsMigration;
+            return isContestedCase(caseDetails) && !hasRegionList(caseData);
         }
         return false;
     }
 
+    private static boolean isContestedCase(CaseDetails caseDetails) {
```

Comment:
Good idea, I would move such methods to some domain class, so that they can accumulate there over time and are available to reuse for further migrations.